openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -290,7 +290,7 @@ def get_template_context(self, user, user_schedules):
             })
 
         if first_schedule is None:
-            self.log_debug('No courses eligible for upgrade for user.')
+            self.log_info('No courses eligible for upgrade for user.')
```

Comment:
I  believe this is logged on a per-recipient basis.  We may want to keep this as debug.